CHANGELOG.md

```diff
@@ -1,3 +1,14 @@
+## 2.2.0
+
+### BUGFIXES
+
```

Comment:
This needs to be the full changelog including everything contained within 2.2.0-rcs

Reply:
Fixed. And mentioning #2335 as `[Cellular] Fix high data usage during poor signal conditions [#2335](https://github.com/particle-iot/device-os/pull/2335)`